fdbkubernetesmonitor/monitor.go

```diff
@@ -152,7 +156,14 @@ func StartMonitor(ctx context.Context, logger logr.Logger, configFile string, cu
 	// Add Prometheus support
 	mux.Handle("/metrics", promHandler)
 	go func() {
-		err := http.ListenAndServe(listenAddr, mux)
+		if promConfig.keyFile != "" || promConfig.certFile != "" {
+			err := http.ListenAndServeTLS(promConfig.listenAddr, promConfig.certFile, promConfig.keyFile, mux)
```

Comment:
We probably rather want to make use of `tls.Config` to allow the fdbkubernetesmonitor to reload the certificate/key file.

Reply:
good point